applications/GeoMechanicsApplication/custom_elements/integration_coefficient_modifier_axisymmetric.cpp

```diff
@@ -0,0 +1,36 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Gennady Markelov
+//                   Anne van de Graaf
+//
+
+#include "integration_coefficient_modifier_axisymmetric.h"
+#include "custom_utilities/element_utilities.hpp"
+
+namespace Kratos
+{
+
+double IntegrationCoefficientModifierForAxisymmetricElement::operator()(double IntegrationCoefficient,
+                                                                        const Geo::IntegrationPointType& rIntegrationPoint,
+                                                                        const Element& rElement) const
+{
+    auto shape_function_values = Vector{};
+    shape_function_values      = rElement.GetGeometry().ShapeFunctionsValues(
```

Comment:
This function returns its result through its return type as well as through the parameter list. Since we're forced to pass the out-parameter anyway, I would suggest to leave out the assignment (since it is redundant). It also makes clear why you need to declare `shape_function_values` upfront.
```suggestion
    rElement.GetGeometry().ShapeFunctionsValues(
```

Reply:
done